renovate.json

```
@@ -8,5 +8,12 @@
   "ignorePaths": [
     ".kokoro/requirements.txt",
     ".kokoro/requirements-aoss.txt"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["*"],
+      "groupName": "all dependencies",
+      "schedule": ["at any time"]
+    }
```

```suggestion
  ]
```
The setup.py and several other files which contain dependency versions are automatically generated. The source of truth for the code files live here.